### DISCUSSION TITLE
How to enable tailwind css auto-completion in template string for Webstorm?

### DISCUSSION THREAD

**Discussion by zeroarst** [ben-rogerson/twin.macro#867] [Q&A]
Hi guys, I just started using this library. It works fine. The only issue that I have is I can't use auto-completion with `tw` template string. The auto-completion does work with normal `className` attribute:
![image](https://github.com/user-attachments/assets/19df48f3-6f94-4761-905a-42c59ea32445)

But with `tw` template string, I can't use it.
![image](https://github.com/user-attachments/assets/544a4db4-8e3d-4852-b2a3-67f55dd38906)

Is it current supported and needed to be configured to enable, or simply just not supported?

Thank you.

**Comment by ben-rogerson** [comment]
Hey there - check out [this comment](https://github.com/ben-rogerson/twin.macro/discussions/227#discussioncomment-479606) on that
